### ISSUE TITLE
Term Request: Moon crater-like corneal endothelial lesions

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11753]
**Preferred term label:**
Moon crater-like corneal endothelial lesions

**Synonyms**
* Moon crater endothelial excavations
* Crater-like corneal endothelial changes
* Endothelial excavations and pits

**Definition (free text, please give PubMed ID)**
Distinct round, excavated, moon crater-like depressions and pits on the posterior surface of Descemet membrane and endothelial layer, best visualized on specular reflection or indirect illumination and associated with focal endothelial cell attenuation (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Abnormal corneal endothelium morphology (HP:0011488)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* X-linked endothelial corneal dystrophy (XECD) (ORPHA:293621, OMIM:300779)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
